metric/instrument.go

```diff
@@ -171,6 +177,21 @@ func (o unitOpt) applyInt64ObservableGauge(c Int64ObservableGaugeConfig) Int64Ob
 // The unit u should be defined using the appropriate [UCUM](https://ucum.org) case-sensitive code.
 func WithUnit(u string) InstrumentOption { return unitOpt(u) }
 
+// WithExplicitBucketBoundaries sets the instrument explicit bucket boundaries.
```

Comment:
Do we want to include some mention about how these are "advice" and may be overridden or ignored by the SDK?

Reply:
nit: "advisory"? 😉 https://github.com/open-telemetry/opentelemetry-specification/pull/3693

Reply:
Added `This option is considered "advisory", and may be ignored by API implementations.`